cmd/operator/main.go

```diff
@@ -67,6 +71,7 @@ func main() {
 		tlsCert     = flag.String("tls-cert-base64", "", "The base64-encoded TLS certificate.")
 		tlsKey      = flag.String("tls-key-base64", "", "The base64-encoded TLS key.")
 		caCert      = flag.String("ca-cert-base64", "", "The base64-encoded certificate authority.")
+		certDir     = flag.String("cert-dir", defaultTLSDir, "The directory which contains TLS certificates for webhook server.")
```

Comment:
You didn't mention this change in the PR description, did you mean opening another PR for this or you meant to essentially implement go/gmp:cert-args for operator? cc @bernot-dev

Reply:
Hm, would't this change break user, if we upgrade image and accidently forgot to change this into existing dir? Are we sure `/etc/tls/private` always exists?

Reply:
It doesn't fully implement go/gmp:cert-args. Versus @bernot-dev's solution this doesn't let you read a pre-existing secret (like the one we'd load from GKE). This is the change mentioned in the PR description about using an emptyDir. Due to the `readOnlyRootFilesystem` flag, we cannot write a self-signed certificate in the root directory anymore, so I settled on mounting the emptyDir to `/etc/tls/private`. As it's mounted this always exists.